stix_shifter_utils/normalization/base_normalization_readme.md

```diff
@@ -0,0 +1,254 @@
+In this STIX Shifter Connector, instead of returning a STIX Observation, we are returning a STIX 2.1 Bundle.
```

Comment:
I would reword this line. This readme is describing a particular type of stix-shifter connector that uses the base normalization class to return a bundle of SDOs. The difference is that the regular json-to-stix flow returns a bundle of STIX 2.1 observed-data objects, where connectors using this normalization class return other SDO.

Reply:
Reworded